index.html

```diff
@@ -2322,6 +2285,68 @@ <h2>Chunk ordering</h2>
 class="chunk">PLTE</span></a> in datastream</figcaption>
 </figure>
 
+</section>
+
+<section id="sec-defining-new-chunks">
+<h2>Defining chunks</h2>
+
+<section id="sec-defining-public-chunks">
+  <h3>General</h3>
+
+  <p>All chunks, private and publicm SHOULD be listed at [[png-reg]].</a>
```

Comment:
I didn't find the original PNG REG list to be helpful; it is simply a concatenation of the chunk types from the PNG spec and the Extensions spec, with no further information. I suppose it will have more value if it also lists the known private chunks, too.

Reply:
Yes, the idea is to have a single place where all chunks are listed, including a link to the defining specification, if available. Otherwise an implementer needs to look through multiple documents.